src/dodal/beamlines/aithre.py

```diff
@@ -0,0 +1,9 @@
+from dodal.common.beamlines.beamline_utils import device_factory
+from dodal.devices.aithre_lasershaping.goniometer import Goniometer
+
+PREFIX = "LA18L-MO-LSR-01:"
+
+
+@device_factory()
+def goniometer() -> Goniometer:
+    return Goniometer(PREFIX, "aithre")
```

Comment:
Should; The second parameter here is the name of the specific device. `aithre` is the name of the whole project but the device can just be called `goniometer` I think